src/data/users-test-data.js

```diff
@@ -3,145 +3,25 @@ const usersData = [
   "name": "Saige O'Kon",
   "id": 1,
   "pantry": [
-    {
```

Comment:
Thank you for cleaning this users test data down. We definitely did not need all of those ingredients elements in user pantry.